src/main/java/it/pagopa/interop/probing/caller/util/ClientUtil.java

```diff
@@ -62,8 +66,9 @@ private TelemetryDto callRest(TelemetryDto telemetryResult, EserviceContentDto s
     long before = System.currentTimeMillis();
     telemetryResult.checkTime(String.valueOf(before));
     AWSXRay.beginSubsegment("Rest_call_to :".concat(service.basePath()[0]));
-    Response response = restClientConfig.feignRestClient()
-        .probing(URI.create(service.basePath()[0]), jwtBuilder.buildJWT(service.audience()));
+    Response response = restClientConfig.feignRestClient().probing(
+        URI.create(service.basePath()[0] + ProjectConstants.PROBING_ENDPOINT_SUFFIX),
```

Comment:
may you remove the potential trailing `/` in the base path?

Reply:
done